utils/Utils.java

```diff
@@ -1,62 +1,54 @@
-package utils;
+package frc.robot.common.utils;
 
 import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
 import edu.wpi.first.math.kinematics.ChassisSpeeds;
 import edu.wpi.first.math.kinematics.SwerveModuleState;
 
 public class Utils {
     public static final double EPSILON = 1e-9;
 
     /**
-     * Epsilon equals with standard epsilon.
+     * sets the value of the joystick to 0 if the value is less than the threshold
+     *
+     * @param val       the joystick value
+     * @param threshold the threshold value
+     * @return 0 if val is less than the threshold else val
      */
+    public static double deadband(double val, double threshold) {
```

Comment:
Perhaps WPILib's [`MathUtil.applyDeadbnd`](https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/math/MathUtil.html#applyDeadband(double,double)) does what you want?